### DISCUSSION TITLE
Seafile Server - Cloud with OnlyOffice or Colabora integration

### DISCUSSION THREAD

**Discussion by BerlinBased** [MichaIng/DietPi#5675] [Software Request] [answered]
Seafile is a very reliable cloud software which works stable. It would be perfect addition for DietPi. There is a official Raspberry Pi version which I couldn't make work with DietPi.

**Comment by Joulinar** [comment, marked as answer]
Basically a duplicate of this one https://github.com/MichaIng/DietPi/discussions/5676

Isn't it?

**Reply by BerlinBased** [reply]
My fault, didn´t found it by search command for any reason.
